src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysQl://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "kodek");
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load MSQL Driver",e);

```

Comment:
The error message in the RuntimeException should be corrected to 'Can't load MySQL Driver' instead of 'Can't load MSQL Driver'. This will make the error message more accurate and helpful for debugging.

Reply:
ok, done